indigo/distribution.yaml

```diff
@@ -1531,6 +1531,25 @@ repositories:
       url: https://github.com/pyros-dev/catkin_pip.git
       version: indigo
     status: developed
+  catkin_virtualenv:
+    doc:
+      type: git
+      url: https://github.com/locusrobotics/catkin_virtualenv.git
+      version: devel
+    release:
+      packages:
+      - catkin_virtualenv
+      - test_catkin_virtualenv
+      - test_catkin_virtualenv_py3
```

Comment:
Do you really want these test packages to be available as Debian packages or should they be ignored in the release step?